src/main/java/com/segment/analytics/android/integrations/adobeanalytics/AdobeIntegration.java

```diff
@@ -85,7 +85,11 @@ public String key() {
               "Video Playback Paused",
               "Video Playback Resumed",
               "Video Content Completed",
-              "Video Playback Completed"));
+              "Video Playback Completed",
```

Comment:
unrelated to this change, but if this is a constant, this variable should be final and named `VIDEO_EVENT_LIST`.

Reply:
I should have caught that - thanks!